src/zope/schema/tests/test_accessors.py

```diff
@@ -59,15 +60,40 @@ def test___provides___w_field_no_provides(self):
     def test___provides___w_field_w_provides(self):
         from zope.interface import implementedBy
         from zope.interface import providedBy
+        from zope.interface.interfaces import IAttribute
+        from zope.interface.interfaces import IMethod
         from zope.schema import Text
+
+        # When wrapping a field that provides stuff,
+        # we provide the same stuff, with the addition of
+        # IMethod at the correct spot in the IRO (just before
+        # IAttribute).
         field = Text()
         field_provides = list(providedBy(field))
         wrapped = self._makeOne(field)
         wrapped_provides = list(providedBy(wrapped))
-        self.assertEqual(wrapped_provides[:len(field_provides)],
-                         list(providedBy(field)))
+
+        index_of_attribute = field_provides.index(IAttribute)
+        expected = list(field_provides)
+        expected.insert(index_of_attribute, IMethod)
+        self.assertEqual(expected, wrapped_provides)
         for iface in list(implementedBy(self._getTargetClass())):
-            self.assertTrue(iface in wrapped_provides)
+            self.assertIn(iface, wrapped_provides)
+
+    def test___provides___w_field_w_provides_strict(self):
+        from zope.interface import ro
+        attr = 'STRICT_IRO'
+        try:
+            getattr(ro.C3, attr)
+        except AttributeError:
+            # https://github.com/zopefoundation/zope.interface/issues/194
```

Comment:
would be nice to briefly say what that issue is about, for offline reading and such